apps/docs/docs/user/composite-block-types.md

```diff
@@ -2,21 +2,24 @@
 sidebar_position: 4
 ---
 
-# Composite Blocks
+# Composite Block Types
 
-Composite blocks are a way to create new blocktypes in Jayvee by combining the functionality of existing blocks and pipes. By relying on composite blocks instead of implementing more builtin blocks in a language interpreter, Jayvee supports easy extension by users.
+_Composite block types_ are a way to create new _block types_ in Jayvee by combining the functionality of existing _blocks_ and _pipes_. By relying on _composite block types_ instead of implementing more _built-in block types_ in a language interpreter, Jayvee supports easy extension by users.
 
-Composite blocks define:
-- with the `property` keyword: properties with a name and [value type](./core-concepts.md#valuetypes), optionally a default value
-- with the `input` keyword: one input with a name and io type (that can be None)
-- with the `output` keyword: one output with a name and io type (that can be None)
-- one pipeline definition, starting from the input (using its name) and ending in the output (again using its name)
-- all blocks that are used in the pipeline definition (either builtin or other composite blocks)
+_Composite block types_ define:
```

Comment:
Nitpicky: I wouldn't change font decoration that much. Do it once when introducing the term but not afterwards. Less differences, easier reading.

I am happy with you leaving this in though if you feel it is better.

Reply:
I think this would be a general thing... 
I worked with the "rule" to make all language concepts italic (_block type_, _pipeline_, ...)
I'd leave it for now, but I'm happy if we decide to change it in the future ;-)